### ISSUE TITLE
Incorrect processorId value in BinaryRecording.m

### ISSUE THREAD

**Issue by vncntprvst** [open-ephys/open-ephys-matlab-tools#3]
Hi,
I get this error when opening a recording session with `session = Session(directory)` - using code v1.0.0 with Matlab 2022b:  
```
Error using  * 
Incorrect dimensions for matrix multiplication. Check that the number of columns in the first matrix matches the number of rows in the second
matrix. To operate on each element of the matrix individually, use TIMES (.*) for elementwise multiplication.

Error in BinaryRecording/loadEvents (line 128)
                self.ttlEvents(id) = DataFrame(abs(lines), sampleNumbers, timestamps, processorId*ones(numEvents,1), repmat(string(id),numEvents,1), lines > 0, ...

Error in BinaryRecording (line 41)
            self = self.loadEvents();

Error in BinaryRecording.detectRecordings (line 254)
                    recordings{end+1} = BinaryRecording(recordingDirectories{recIdx}, expIdx, recIdx);

Error in RecordNode/detectRecordings (line 95)
                self.recordings = BinaryRecording.detectRecordings(self.directory);

Error in RecordNode (line 71)
            self.detectRecordings();

Error in Session/detectRecordNodes (line 70)
                self.recordNodes{end+1} = RecordNode(paths{i});

Error in Session (line 61)
            self.detectRecordNodes();
```
The `processorId` variable is empty, from its definition at [line 117](https://github.com/open-ephys/open-ephys-matlab-tools/blob/879bec21fa13cc8e4f8d9c56b530a049b0b57c1f/open_ephys/analysis/formats/BinaryRecording/BinaryRecording.m#L117).
At this stage, `fullId`'s value is `1×2 cell array {'DAQmx-109'}    {'PXIe-6341'}`, which leads `str2num` to return an empty value. 
What is the expected behavior of `processorId = str2num(fullId{1});` on line 117 ?

**Comment by medengineer** [comment]
That _should_ return just the processor ID, in your case 109. 

The issue is the '-' in 'NI-DAQmx' that's throwing off the indexing when splitting on '-' to get the various parts. 

Will fix this with a general solution shortly, thanks for catching this.

**Comment by medengineer** [comment]
Fixed in [v1.0.1](https://github.com/open-ephys/open-ephys-matlab-tools/releases/tag/v1.0.1)

**Comment by vncntprvst** [comment]
It works now! 
Thank you
